module_utils/foreman_helper.py

```diff
@@ -120,9 +124,13 @@ def find_resource(self, resource, search, failsafe=False, thin=False):
                 result = self.show_resource(resource, result['id'])
         return result
 
-    def find_resource_by_name(self, resource, name, failsafe=False, thin=False):
+    def find_resource_by_name(self, resource, name, organization_id=None, failsafe=False, thin=False):
```

Comment:
We may want to allow passing in other params, for exmaple https://theforeman.org/plugins/katello/3.9/api/apidoc/v2/repository_sets/index.html requires `product_id`, I'm fairly sure there are others.

But I think we can change this at a later date when/if we require them.

Reply:
Yea, maybe this could be done with a dict parameter or with kwargs.

Reply:
Yeah, this is something I should change before merging!
Otherwise fixing the API later is pain.

Reply:
updated to be a dict.